### DISCUSSION TITLE
Mention a user in comments

### DISCUSSION THREAD

**Discussion by Lanhild** [growilabs/growi#8791] [Ideas / Requests]
Being able to mention another user in the comments would be a neat feature.

**Comment by yuki-takei** [comment]
[It had been implemented once](https://github.com/weseek/growi/pull/5856), but it has been omitted.
Maybe we should consider re-implementing it.

**Reply by Lanhild** [reply]
It would make collaborative editing better, yes.

**Comment by yuki-takei** [comment]
User mentions in comments have been supported since [v5.0.6](https://github.com/growilabs/growi/releases/tag/v5.0.6). You can mention users by typing `@username` in a comment, and they will receive a notification.

Thank you for your suggestion!
